### ISSUE TITLE
Inlay hint filtering is incorrect when there is a newline after the (

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#9606]
### Environment

- OS and Version:
- VS Code Version:
- C/C++ Extension Version:
- Other extensions you installed (and if the issue persists after disabling them):  
- If using SSH remote, specify OS of remote machine:  
- A clear and concise description of what the bug is, including information about the workspace (i.e. is the workspace a single project or multiple projects, size of the project, etc).

### Bug Summary and Steps to Reproduce

A function call with a newline after the ( isn't getting the inlay hint hiding parsed correctly.
![image](https://user-images.githubusercontent.com/19859882/179871002-e97e5153-02b2-4824-9333-0cfca9951b6a.png)
the first param "i" should be filtering out as a match.

### Code sample and Logs

```cpp
void func(int i, int j) {}

void func2()
{
    int i, j;
    func(
        i,
        j
    );
}
```

**Comment by michelleangela** [comment]
This also reproduces on VS.

Sample code:

```

int volume(const int depth, const int width, const int height)
{
	return width * height;
}

void test()
{
	int a = 0;
	a = volume(5 /*depth*/, 4 /*width*/, 3);
	a = volume( 5 /*depth*/, 4 /*width*/, 3);
	a = volume(  5 /*depth*/, 4 /*width*/, 3);
	a = volume(   5 /*depth*/, 4 /*width*/, 3);
	a = volume(
				    5 /*depth*/,
			   4 /*width*/,
				3);
}
```

**Comment by sean-mcmanus** [comment]
@michelleangela I'm not reproing the bug in VS -- are you using the latest dogfood version and the "hide" settings?

**Comment by michelleangela** [comment]
There needs to be three spaces before the `5` in `a = volume(   5 /*depth*/, 4 /*width*/, 3);`

**Comment by sean-mcmanus** [comment]
Yeah, I repro that now, strange that it only repros after there's 3 or more spaces and how the 5th case that comes after it (and my original repro cases) doesn't repro in VS but repros in VS Code.

**Comment by sean-mcmanus** [comment]
Fixed with 1.12.0 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.12.0
